software/solidtime.yml

```
@@ -0,0 +1,10 @@
+name: "solidtime"
+website_url: "https://www.solidtime.io"
+source_code_url: "https://github.com/solidtime-io/solidtime"
+description: "solidtime is a modern time tracking application for freelancers and agencies."
```

```suggestion
description: "Modern time tracking application for freelancers and agencies."
```

https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines